File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
import java.util.Arrays;
import java.util.List;

class Solution {
    public int removeDuplicates(int[] nums) {
        Set<Integer> intSet = new LinkedHashSet<>();
        int i = 0;
        for (int num : nums) {
            //System.out.println(num);

            intSet.add(num);
        }
        int k = intSet.size();

        for (int loopInt : intSet) {
            nums[i] = loopInt;
            System.out.println(i);
            i++;
        }
        //System.out.println(nums);
        return k;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.go
func removeDuplicates(nums []int) int {
    lastUnique := 0
    for i := 1; i < len(nums); i++ {

        if nums[i-1] != nums[i] {
            nums[lastUnique+1] = nums[i]
            lastUnique++
        }
    }

    return lastUnique+1
}